src/App.tsx

```diff
@@ -1,63 +1,111 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+enum SortType {
+  NAME = 'name',
+  LENGTH = 'length',
+}
+
+const goodsFromServer = [
```

Comment:
you can create a type for goods
```
type Good = {
  id: number;
  name: string;
}
```
```suggestion
const goodsFromServer: Good[] = [
```